README.md

```diff
@@ -1,28 +1,33 @@
-# authzed dotnet
-[![](https://img.shields.io/badge/discord-spicedb-7289da?style=flat-square "Discord Badge")](https://discord.gg/spicedb)
-[![](https://img.shields.io/badge/twitter-@authzed-1d9bf0.svg?style=flat-square "Twitter Badge")](https://twitter.com/authzed)
-[![](https://img.shields.io/badge/linkedin-+authzed-2D65BC.svg?style=flat-square "LinkedIn Badge")](https://linkedin.com/company/authzed)
+# Authzed.Net
+[![Nuget](https://img.shields.io/nuget/v/Authzed.Api?color=%23006dad)](https://www.nuget.org/packages/Authzed.Net)
+[![License](https://img.shields.io/badge/license-Apache--2.0-blue.svg)](https://www.apache.org/licenses/LICENSE-2.0.html)
+[![Build Status](https://github.com/authzed/authzed-dotnet/workflows/Test/badge.svg)](https://github.com/authzed/authzed-dotnet/actions)
+[![Mailing List](https://img.shields.io/badge/email-google%20groups-4285F4)](https://groups.google.com/g/authzed-oss)
+[![Discord Server](https://img.shields.io/badge/discord-spicedb-7289da?style=flat-square "Discord Badge")](https://discord.gg/spicedb)
+[![Twitter](https://img.shields.io/badge/twitter-@authzed-1d9bf0.svg?style=flat-square "Twitter Badge")](https://twitter.com/authzed)
+[![LinkedIn](https://img.shields.io/badge/linkedin-+authzed-2D65BC.svg?style=flat-square "LinkedIn Badge")](https://linkedin.com/company/authzed)
 
 This repository houses the official .NET (generated in C#) client library for Authzed and SpiceDB.
 
-This project is intended to be a canonical source of generated ProtoBuf messages for gRPC communication.
+[Authzed] is a database and service that stores, computes, and validates your application's permissions.
```

Comment:
It's present in the [authzed-py](https://github.com/authzed/authzed-py/blob/main/README.md?plain=1#L12) repo as well, which is what I was cribbing from. It read strangely to me as well, but I figured it might be related to SEO on the READMEs. I can ask @jzelinskie though.

Reply:
We can land this and fix it after the fact if needed.